content/en/tracing/livetail.md

```diff
@@ -23,49 +23,44 @@ further_reading:
   text: "Analytics on your APM data at infinite cardinality"
 ---
 
-{{< img src="tracing/live_tail/livetail_view.gif" alt="Live tail" >}}
+{{< img src="tracing/live_tail/livesearchmain.gif" alt="Live Search" >}}
 
 ## Overview
 
-The APM [Live Tail][1] gives you the ability to see all trace spans in near real-time from anywhere in the Datadog UI. It displays spans as soon as they get out of the Datadog Agent section and before they are indexed by Datadog. All trace spans ingested by Datadog are displayed (Tracing without Limits\*) after being processed. With the APM Live tail page you can:
+The APM [Live Search][1] gives you the ability to search all ingested spans using any tag in real-time for the past 15 minutes. It displays spans as soon as they are sent by the Datadog agent and before they are indexed by Datadog. All spans ingested by Datadog are displayed without any sampling (Tracing without Limits\*). With the APM Live Search you can:
```

Comment:
```suggestion
The APM [Live Search][1] gives you the ability to search all ingested spans using any tag in real-time for the past 15 minutes. It displays spans as soon as they are sent by the Datadog Agent and before they are indexed by Datadog. All spans ingested by Datadog are displayed without any sampling (Tracing without Limits\*). With the APM Live Search you can:
```

Reply:
Is `(Tracing without Limits\*)` supposed to be a side note or a link? The \* seems odd here.